build-system/server/app.js

```diff
@@ -1225,7 +1225,7 @@ app.get(
     }
     const isJsMap = filePath.endsWith('.map');
     if (isJsMap) {
-      filePath = filePath.replace(/\.(js|mjs)\.map$/, '.$1');
+      filePath = filePath.replace(/\.(m?js)\.map$/, '.$1js');
```

Comment:
```suggestion
      filePath = filePath.replace(/\.(m?js)\.map$/, '.$1');
```

Here and elswhere, since `"js"` are now a part of the capture group

Reply:
fixed. thanks for being thorough